modules/installation-cis-ibm-cloud.adoc

```diff
@@ -1,17 +1,35 @@
 // Module included in the following assemblies:
 //
 // installing/installing_ibm_cloud_public/installing-ibm-cloud-account.adoc
+// installing/installing_ibm_power/installing-ibm-cloud-account-power-vs.adoc
+
+ifeval::["{context}" == "installing-ibm-cloud-account-power-vs"]
+:ibm-power-vs:
+endif::[]
 
 :_content-type: PROCEDURE
 [id="installation-cis-ibm-cloud_{context}"]
 = Configuring DNS resolution using Cloud Internet Services
 
+ifndef::ibm-power-vs[]
 IBM Cloud Internet Services (CIS) is used by the installation program to configure cluster DNS resolution and provide name lookup for the cluster to external resources. Only public DNS is supported with IBM Cloud VPC.
+endif::ibm-power-vs[]
+ifdef::ibm-power-vs[]
+IBM Cloud Internet Services (CIS) is used by the installation program to configure cluster DNS resolution and provide name lookup for the cluster to external resources. Only public DNS is supported for IBM Power VS IPI.
+endif::ibm-power-vs[]
 
+ifndef::ibm-power-vs[]
 [NOTE]
 ====
 IBM Cloud VPC does not support IPv6, so dual stack or IPv6 environments are not possible.
 ====
+endif::ibm-power-vs[]
+ifdef::ibm-power-vs[]
```

Comment:
Is IPv6 only supported with UPI for IBM Power in 4.11?

Reply:
correct. we use VPCs from IBM Cloud